receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -215,15 +213,10 @@ func (jr *jReceiver) Start(_ context.Context, host component.Host) error {
 }
 
 func (jr *jReceiver) Shutdown(context.Context) error {
-	jr.mu.Lock()
-	defer jr.mu.Unlock()
-
-	return jr.stopTraceReceptionLocked()
-}
-
-func (jr *jReceiver) stopTraceReceptionLocked() error {
 	var err = componenterror.ErrAlreadyStopped
 	jr.stopOnce.Do(func() {
```

Comment:
Doesn't need to block this PR but Is this stopOnce needed? I believe we decided that collector guarantees Shutdown is only called once. Right now there are a mixture of components with stopOnce so we should probably get rid of all of them if not needed.

Reply:
Would address in a separate PR.

Reply:
Given you removed `mu` from `Shutdown` in this PR, I would recommend removing this from `Start` as well (and may as well remove `startOnce` / `stopOnce` as well while you're at it 😄 )

Reply:
Did I? It is just moved the next line.